### ISSUE TITLE
NVDA does not announces complete information after user Input “3000” as  local port and press enter

### ISSUE THREAD

**Issue by tripathiakhil429** [microsoft/vscode#143278]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**GitHub Tags:**#A11yMAS;WCAG1.3.1;A11ySev2;#A11yTCS;#VisualStudioCode-Win32-Apr2021;#Visual Studio Code Client;#DesktopApp;#Win10;#NVDA
### Actual:
When NVDA is on and user Input “3000” as  local port and press enter, NVDA does not announces complete information. It only announces "Quick Input list Launch via NPM 1 of 1" and does not announces name of combo box.
Similar issue is observed with JAWS

### Expected:

When NVDA is on and user Input “3000” as  local port and press enter, NVDA should announce complete information. It should announce combobox chosse the launch configuration to use to run tour component locally  Quick Input list Launch via NPM 1 of 1

### Environment Details:
Visual studio code
kubernetes
Windows Version: Windows10
Screen reader: NVDA(2020.2), Jaws(2020.2008.24)
### Repro Steps:

1. Setup The Kubernetes extension.
2. Open visual studio code and load "Bikes" project.
3. Tab to "View" menu item in top navigation bar, press enter to expand and use down arrow key to Select "Command Pallet" option.
4. Search and select "Local Process with Kubernetes: Configure Local Process with Kubernetes”
5. Connect to Kubernetes panel will open.
6. Tab to bike options and press enter
7. Start the NVDA 
8. Input “3000” as your local port and press enter to check the result

**Comment by alexr00** [comment]
@tripathiakhil429 can you share a screen shot or recording of what's happening?
